storybook/main.js

```diff
@@ -74,6 +74,24 @@ module.exports = {
 	docs: {},
 	typescript: {
 		reactDocgen: 'react-docgen-typescript',
+		// Should match defaults in Storybook except for the propFilter.
+		// https://github.com/storybookjs/storybook/blob/3e34a288c8fabc7d5b5cc43b28ae9d674c48e3ea/code/core/src/core-server/presets/common-preset.ts#L162-L168
+		reactDocgenTypescriptOptions: {
+			shouldExtractLiteralValuesFromEnum: true,
+			shouldRemoveUndefinedFromOptional: true,
+			propFilter: ( prop ) => {
+				if ( ! prop.parent ) {
+					return true;
+				}
+
+				if ( /@base-ui|@ariakit/.test( prop.parent.fileName ) ) {
+					return true;
+				}
```

Comment:
This is so prop data coming from our underlying libraries will also show up in the props tables.